erreser.go

```
@@ -0,0 +1,624 @@
+		Title:  feedData.Image.Title,
+		FeedId: int(existingFeed.ID),
+	}
+	db.Where(&models.Image{FeedId: int(existingFeed.ID)}).FirstOrCreate(&models.Image{}, image)
+
+}
+
+func reInitFeeds() {
+	log.Println("Reinitializing feeds")
+	dbParams := database.InitDbParams()
+	db := database.DbConnect(dbParams)
+	db.AutoMigrate(&models.Feed{})
```

golangci-lint (1.62.2)

Error return value of `db.AutoMigrate` is not checked

(errcheck)